docker/Dockerfile.dev

```diff
@@ -0,0 +1,17 @@
+# To build this container, go to ESMValTool root folder and execute:
+# docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile.dev
+FROM continuumio/miniconda3
+
+COPY . /src/ESMValTool
+WORKDIR /src/ESMValTool
+
+RUN conda update -y conda pip && conda env update --name base --file environment.yml && conda clean --all -y
+RUN pip install . && pip cache purge
+RUN apt install julia -y && julia esmvaltool/install/Julia/setup.jl
```

Comment:
To fix the julia diagnostics, I think you need to run the installed version of the script here, not the one from the source code. Or install the tool in editable mode with `pip -e`.

Reply:
Can you test again, please?